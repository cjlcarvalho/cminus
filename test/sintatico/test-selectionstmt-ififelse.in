void main(void){
	if(a==b)
		return;
	if(a>=b)
		a = a+b;
	else{
		return;
	}
	return;
}
