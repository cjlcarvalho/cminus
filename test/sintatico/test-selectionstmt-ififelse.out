[program 
  [fundecl 
    [void]
    [main]
    [paramlist]
    [block 
      [if 
        [== [var [a]] [var [b]]]
        [return ]
      ]
      [if 
        [>= [var [a]] [var [b]]]
        [= [var [a]] 
          [+ [var [a]] [var [b]]]]
        [block 
          [return ]
        ]
      ]
      [return ]
    ]
  ]
]
